### PR TITLE
liquidprompt HEAD, devel

### PR DESCRIPTION
**Pull request by augmentedfourth**
Moving HEAD into its own block & adding a separate devel block

### CODE REVIEW

**Comment by augmentedfourth** [comment]
The project has changed its structure & now maintains head/master as separate from devel/develop.

File: Formula/liquidprompt.rb [hunk 1]
```diff
@@ -3,10 +3,19 @@ class Liquidprompt < Formula
   homepage "https://github.com/nojhan/liquidprompt"
   url "https://github.com/nojhan/liquidprompt/archive/v_1.10.tar.gz"
   sha256 "5a1e01b86e81002ac20c61cbd96f5fd276db7f3fa09ef220f3db0fb7aa614e24"
-  head "https://github.com/nojhan/liquidprompt.git", :branch => "develop"
 
   bottle :unneeded
 
+  head do
```
Comment (DomT4) [inline Formula/liquidprompt.rb]: This doesn't need moving, or the explicit `version`. You just need to remove the `:branch => "develop"` from the original line here.

File: Formula/liquidprompt.rb [hunk 2]
```diff
@@ -3,10 +3,19 @@ class Liquidprompt < Formula
...
+    url "https://github.com/nojhan/liquidprompt.git", :branch => "master"
+    version "head"
+  end
+
+  devel do
```
Comment (DomT4) [inline Formula/liquidprompt.rb]: We don't accept new `devel` blocks that aren't pinned to a tarball or specific git tag/revision. We plan to eventually bottle `devel` stuff and we can't do that if they can move around freely.

**Comment by augmentedfourth** [comment]
fixed according to suggestions from @DomT4.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/b89da345b00c9ee69c62e5307371185603153b2d. Thank you for your contribution to Homebrew @augmentedfourth; we appreciate it! :smiley_cat:

**Pull request merged**